lib/utils/report.mjs

```diff
@@ -32,11 +45,11 @@ export default function report(problem) {
 		return;
 	}
 
-	const { start } = (node && node.rangeBy({ index, endIndex })) || {};
+	const reportRange = (node && node.rangeBy({ index, endIndex })) || {};
```

Comment:
Since node is required I am gonna assume that `report` as an util is being used in the wild without passing a node.
i.e. removing `node &&` is too risky